include/net/nrf_cloud.h

```diff
@@ -955,6 +964,23 @@ bool nrf_cloud_fota_is_type_modem(const enum nrf_cloud_fota_type type);
  * @retval false Specified FOTA type is not enabled by the configuration.
  */
 bool nrf_cloud_fota_is_type_enabled(const enum nrf_cloud_fota_type type);
+
+/**
+ * @brief Start the FOTA update job.
+ *        This function should be called after the NRF_CLOUD_EVT_FOTA_JOB_AVAILABLE event
+ *        is received.
+ *        This function should only be called if @kconfig(CONFIG_NRF_CLOUD_FOTA_AUTO_START_JOB)
+ *        is disabled.
+ *        Depends on @kconfig(CONFIG_NRF_CLOUD_FOTA).
+ *
+ * @retval 0            FOTA update job started successfully.
+ * @retval -ENOTSUP     Error; @kconfig(CONFIG_NRF_CLOUD_FOTA_AUTO_START_JOB is enabled).
```

Comment:
should "is enabled" be outside of the close parenthesis?

Reply:
😅 yes thank you... probably why the doc build was mysteriously failing.